includes/Integrations/Core_Themes_Support.php

```diff
@@ -77,9 +105,29 @@ public function extend_theme_support() {
 	 */
 	public function embed_web_stories() {
 		$customizer = new Customizer();
+		$this->enqueue_style( $this->style_handle );
 		echo $customizer->render_stories(); // phpcs:ignore -- WordPress.Security.EscapeOutput.OutputNotEscaped - Escaped web stories HTML.
 	}
 
+	/**
+	 * Add a class if it is one of supported core themes.
+	 *
+	 * @since 1.3.0
+	 * @param array $classes Array of body classes.
+	 *
+	 * @return array Updated array of classes.
+	 */
+	public function add_core_theme_classes( $classes ) {
+
+		$classes[] = 'has-web-stories-support';
```

Comment:
Do we really need this?

Reply:
This was to tell the theme that current theme has support added for the web stories, just in case devs would want. There is no direct use case right now.

Reply:
Then we should probably add this to all themes that declare theme support, not just core themes?

But since we don't have a use case for this yet, I think we can remove it for now.

Reply:
also some core theme elements also need to be adjusted based on whether the stories are showing or not. like pushing down the content a bit or removing top margin in case of `2014`. without this class we won't be able to target them.

Reply:
just keeping `.is-showing-header-stories` class to update the page elements where needed ( i.e., 2014 ). But now it uses plain selectors no more `>` needed that too this class will only get added if the stories are showing.

Reply:
> also some core theme elements also need to be adjusted based on whether the stories are showing or not.

Ah, good to know! Makes sense to keep then.

Perhaps we can rename the class name to `has-web-stories` or `is-displaying-web-stories`